src/search/include/search/adapter.h

```diff
@@ -26,7 +27,12 @@ template <typename Plant,
 class get_next_canonical_words
 {
 public:
-	get_next_canonical_words(const std::set<ActionType> & = {}, const std::set<ActionType> & = {})
+	/** Construct the comparator with the given action partitioning.
+	 * @param controller_actions The actions that the controller can select
+	 * @param environment_actions The actions that the environment can select
+	 */
+	get_next_canonical_words([[maybe_unused]] const std::set<ActionType> &controller_actions  = {},
+	                         [[maybe_unused]] const std::set<ActionType> &environment_actions = {})
```

Comment:
Should this warning be addressed in this PR or in a separate one?

Reply:
This warning can be ignored, the function body is empty anyway, as this is just the base class.